src/engraving/rw/400/twrite.cpp

```diff
@@ -198,6 +198,9 @@ class WriteVisitor : public rtti::Visitor<WriteVisitor>
 void TWrite::writeItem(const EngravingItem* item, XmlWriter& xml, WriteContext& ctx)
 {
     bool found = WriteVisitor::visit(WriteTypes {}, item, xml, ctx);
+#ifdef NDEBUG // avoid compiler warning
+    UNUSED(found);
+#endif
     DO_ASSERT(found);
```

Comment:
Thanks, but I already fixed it, I improved the `DO_ASSERT` macro, 
 (the fix will be in my next PR)

Reply:
https://github.com/musescore/MuseScore/pull/17365/commits/9f33cede7c3a4da5f4ff7fc1c2440c895e57266c

Reply:
> (Wouldn't it be wise to have that UNUSED() inside these Q_ASSERT() macros?)

Exactly!

Reply:
I tried the same, but got tons of compiler errors (when `cond` is something like `value = function()`)

Reply:
I think `value = function()` should not be, maybe just `function()`

Reply:
Ah, it is gone, guess I didn't do a `make clean` before.